pkg/reconciler/openshift/tektonaddon/tektonaddon.go

```diff
@@ -280,6 +301,28 @@ func (r *Reconciler) checkComponentStatus(ctx context.Context, labelSelector str
 	return nil
 }
 
+func (r *Reconciler) ensurePAC(ctx context.Context, ta *v1alpha1.TektonAddon) error {
+	pacManifest := r.manifest
```

Comment:
```suggestion
	pacManifest := mf.Manifest{}
```

at present `r.manifest`is empty. But if that changes it could break this function